### DISCUSSION TITLE
Validations rules should receive context

### DISCUSSION THREAD

**Discussion by OrkhanAlikhanov** [goravel/goravel#298] [Ideas]
Rules that make database requests or similar like https://github.com/goravel/example/blob/3c93f6fa07743bc9812c8a61e7ec452ed32b94c2/app/rules/exists.go should receive context.

**Comment by hwbrzzl** [comment]
Good idea, will create an issue for this.

**Reply by hwbrzzl** [reply]
https://github.com/goravel/goravel/issues/303
